Explorer/Assets/DCL/ExplorePanel/ExplorePanelController.cs

```diff
@@ -47,23 +52,29 @@ public ExplorePanelController(ViewFactoryMethod viewFactory,
             NavmapController navmapController,
             SettingsController settingsController,
             BackpackController backpackController,
+            CameraReelController cameraReelController,
             ProfileWidgetController profileWidgetController,
             ProfileMenuController profileMenuController,
             DCLInput dclInput,
+            IExplorePanelEscapeAction explorePanelEscapeAction,
             INotificationsBusController notificationBusController,
             IMVCManager mvcManager,
-            IInputBlock inputBlock)
+            IInputBlock inputBlock,
+            bool includeCameraReel)

```

Comment:
You are right, it is not clean but as you correctly said it will be removed once the feature is released. It's needed just to hide it within the releases (using a feature flag was considered but it was found to be messy for testing) still being able to merge PRs